virttest/qemu_devices/qcontainer.py

```diff
@@ -758,13 +758,28 @@ def hook_fill_scsi_hbas(self, params):
                                                                           'pci.0')},
                                                    child_bus=bus))
 
+    def root_type_list(self):
```

Comment:
Sorry for not noticing this earlier, but this name is not really well chosen. It's part of `qcontainer` and `root_type` would most probably mean something like type of main-system-bus and not the list of acceptable pcie root-ports. Anyway I'm in doubt whether it's actually necessary to be set here. I don't think it's necessary later in the code (I'll put the reasons inline...).